crunchyroll/segment-singular-ios
# SDK preparations and fixes

### Background
In order to use singular-segment-ios integration, it's required to fix some of the framework referencing  issues happening because of invalid includes in SingularIntegration.m

### What has been done?
1. Fixed integration to reference correctly Singular framework
2. Added PR template for future use

### How to test?
1. Add pod 'Segment-Singular', :git => 'git@github.com:crunchyroll/segment-singular-ios.git', :branch => 'CRGROW-1682-sdk-preparations-and-fixes' to the PodFile of the project.
2. Make sure it's compiled successfully
